permissions/plugin-ca-apm.yml

```diff
@@ -3,5 +3,7 @@ name: "ca-apm"
 github: "jenkinsci/ca-apm-plugin"
 paths:
 - "org/jenkins-ci/plugins/ca-apm"
+#added new developer
```

Comment:
Huh?

This is the job of the SCM's commit history.

Reply:
It was added just an attempt from the submitter to nudge the PR build, FWIW. I should have removed it myself before merging.